maxim/logger/livekit/module_functions.py

```
@@ -0,0 +1,109 @@
+
+        else:
+            return False
+
+    except ImportError:
+        return False
+    except Exception:
+        return False
+
+
+# Keep the old function name for backward compatibility but make it call the new one
+def instrument_module_functions():
```

Ruff (0.11.9)

Missing return type annotation for public function `instrument_module_functions`

Add return type annotation: `None`

(ANN201)